scripts/azure/storage-manager.sh

```diff
@@ -0,0 +1,138 @@
+
+CONTAINERS=$( { echo "$buckets_sc"; echo "$buckets_wc"; } | sort | uniq | tr '\n' ' ' | sed s'/.$//')
+
+function create_resource_group() {
+
+    echo "checking if resource group exists" >&2
+    GROUP_EXISTS=$(az group list --query '[].name' | awk "/${CK8S_ENVIRONMENT_NAME}-storage-resource-group/")
+    if [ "$GROUP_EXISTS" ]; then
+        echo "resource group [${CK8S_ENVIRONMENT_NAME}-storage-resource-group] already exists" >&2
+        echo "continue using this group ? (y/n)" >&2
+        read -r -n 1 cmdinput
+        case "$cmdinput" in
+        y|Y) return ;;
+        *) exit 0 ;;
+        esac
+    else
+        echo "resource group [${CK8S_ENVIRONMENT_NAME}-storage-resource-group] does not exist, creating it now" >&2
+        az group create \
+            --name "$CK8S_ENVIRONMENT_NAME"-storage-resource-group \
+            --location "${AZURE_LOCATION}" --only-show-errors
+    fi
+
+}
+
+function create_storage_account() {
+
+    echo "checking if storage account exists" >&2
+    ACCOUNT_EXISTS=$(az storage account list --query '[].name' | awk "/${CK8S_ENVIRONMENT_NAME}storageaccount/")
+    if [ "$ACCOUNT_EXISTS" ]; then
+        echo "storage account [${CK8S_ENVIRONMENT_NAME}storageaccount] already exists" >&2
+        echo "contnue using this account ? (y/n)" >&2
+        read -r -n 1 cmdinput
+        case "$cmdinput" in
+        y) return ;;
+        n) exit 0 ;;
+        esac
+    else
+        az storage account create \
+            --name "$CK8S_ENVIRONMENT_NAME"storageaccount \
+            --resource-group "$CK8S_ENVIRONMENT_NAME"-storage-resource-group \
+            --location swedencentral \
+            --sku Standard_RAGRS \
+            --kind StorageV2 \
+            --allow-blob-public-access false --only-show-errors
+    fi
+}
+
+function create_containers() {
+
+    CONTAINERS_LIST=$(az storage container list --account-name "$CK8S_ENVIRONMENT_NAME"storageaccount --query '[].name' --only-show-errors)
+
+    # shellcheck disable=SC2068
+    for container in ${CONTAINERS[@]}; do
+
+        echo "checking status of container ${container}]" >&2
+
+        CONTAINER_EXISTS=$(echo "$CONTAINERS_LIST" | awk "/${container}/")
+
+        if [ "$CONTAINER_EXISTS" ]; then
+            echo "container ${container}] already exists, do nothing" >&2
+        else
+            echo "container ${container}] does not exist, creating it now" >&2
+            az storage container create \
+                -n "$container" \
+                --account-name "$CK8S_ENVIRONMENT_NAME"storageaccount --only-show-errors
+        fi
+    done
+}
+
+function delete_all() {
+    az group delete --name "$CK8S_ENVIRONMENT_NAME"-storage-resource-group
+}
+
+if [[ "$ACTION" == "$CREATE_ACTION" ]]; then
+    echo "Creating Resource Group" >&2
+    create_resource_group
+
+    echo "Creating Storage Account" >&2
+    create_storage_account
+
+    echo "Creating Storage Containers"
```

Comment:
I assume this should have 
```suggestion
    echo "Creating Storage Containers" >&2
```